src/main/resources/banner.txt

```diff
@@ -0,0 +1,7 @@
+  ______                _
```

Comment:
Mb remove that and choose @Prosperro-de version.
That version is not working as expected